Update Get-TlsCipherSuite.md (#2485)
MicrosoftDocs/windows-powershell-docs: docset/winserver2019-ps/tls/Get-TlsCipherSuite.md

# Get-TlsCipherSuite

## SYNOPSIS

> Gets the cipher suites for TLS for a computer.

Reviewer: Could this be rephrased to "Gets the list of TLS cipher suites available on the computer."?

Author: My thinking here is that the command returns a collection of objects. A list or table is the result of default formatting and not the command.. So the command gets the cypher suites. 

At least that is why I made the change.

> Gets the cipher suites for TLS for a computer.

Reviewer 2: ```suggestion
Gets the list of TLS cipher suites available on the computer.
```

Reviewer 2: @doctordns - please commit the suggested change as per @tseward's request. Many thanks.

Protocols             : {771, 65277}
```

This command gets all the cipher suites that have names that contain the string `AES`.
Note that the name match is case sensitive and this command returns no output for the name `aes`.

> The output includes a field for the TLS/SSL protocols supported by the cipher. See [Cipher Suites in TLS/SSL (Schannel SSP)](https://docs.microsoft.com/windows/desktop/secauthn/cipher-suites-in-schannel) for more information.

Reviewer 2: ```suggestion
The output includes a field for the TLS/SSL protocols supported by the cipher. See [Cipher Suites in TLS/SSL (Schannel SSP)](/windows/desktop/secauthn/cipher-suites-in-schannel) for more information.
```
